### DISCUSSION TITLE
3D wave field

### DISCUSSION THREAD

**Discussion by FURUNZ** [SeisSol/SeisSol#1012] [General]
How to get 3D stress wave calculation results, such as 3D stress field and displacement field results, can output the xdmf file of the whole study domain? Setting &Output Format = 1,5,10 in the parameter.par file has an impact on this?

**Comment by vikaskurapati** [comment]
If you choose `iOutputMask = 1 1 1 1 1 1 1 1 1` under &Output, it outputs all the 3D stresses and velocities in the study domain if you don't give any `OutputRegionBounds`. You then need to integrate the velocities to obtain the displacement field results as of now. The Format under &Output is to choose the format of the output file to be written and 6 sets it to hdf5 output.

Refer to: https://seissol.readthedocs.io/en/latest/parameter-file.html for more details on different available parameters in parameters file
